tests/postgres_tests/test_ranges.py

```diff
@@ -50,6 +50,39 @@ class Model(PostgreSQLModel):
                 instance = Model(field=value)
                 self.assertEqual(instance.get_field_display(), display)
 
+    def test_discrete_range_fields_invalid_default_bounds_argument(self):
+        discrete_range_fields = [
+            pg_fields.IntegerRangeField,
+            pg_fields.BigIntegerRangeField,
+            pg_fields.DateRangeField
+        ]
+
+        for discrete_range_field in discrete_range_fields:
+            field_class_name = f'{discrete_range_field.__class__.__name__}'
+            with self.assertRaises(TypeError, msg=f"Cannot use 'default_bounds' with {field_class_name}"):
```

Comment:
I'm used with pytest parametrization for such cases. Is there a recommend code-style regarding that ?

Reply:
```suggestion
            msg = f"Cannot use 'default_bounds' with {field_class_name}."
            with self.assertRaisesMessage(TypeError, msg):
```